src/main/kotlin/com/icat/orboarding/user/adapters/configuration/DependencyInjection.kt

```diff
@@ -0,0 +1,14 @@
+package com.icat.orboarding.user.adapters.configuration
```

Comment:
Man, dúvida de arquitetura:

Quando eu for criar as injeções da parte de segurança, posso colocar elas aqui tbm? Ou seria uma boa prática criar um `SecurityDependencyInjection` pra separar o contexto de segurança e do domínio da aplicação?

N sei se deu pra entender minha dúvida rsrs

Reply:
Acredito que pode colocar na mesma classe por enquanto maninho, se a gente ver que vai começar a virar uma bagunça, podemos refatorar e deixar uma classe de DI pra cada contexto.